modules/acquia_cms_common/acquia_cms_common.module

```diff
@@ -32,6 +33,24 @@ function acquia_cms_common_module_preinstall($module) {
   if ($module == 'acquia_cms_common') {
     _acquia_cms_common_rewrite_configuration('workflows.workflow.editorial', 'acquia_cms_common');
   }
+  // Create roles if not already available in the system
+  // during content models module getting installed.
+  $content_model_modules = [
+    'acquia_cms_article',
+    'acquia_cms_event',
+    'acquia_cms_page',
+    'acquia_cms_person',
+    'acquia_cms_place',
+  ];
+  if (in_array($module, $content_model_modules)) {
+    $entityTypeManager = \Drupal::entityTypeManager();
+    $roles = get_content_model_roles();
+    foreach ($roles as $role => $role_config) {
+      if (empty($entityTypeManager->getStorage('user_role')->load($role))) {
```

Comment:
Can we use Role::load($role)) here to check the existence of roles? As we are using Role::create method to create the roles.
This will also remove the code from line#46

Reply:
Thank you @apathak18, I have update the code as per suggestion.